### PR TITLE
Add the 34 orphaned Android SDK class & interface articles to the Bing Maps SDK TOC

### PR DESCRIPTION
**Pull request by stevemunk**

### CODE REVIEW

File: BingMaps/sdk-native/TOC.yml [hunk 1]
```diff
@@ -74,6 +74,126 @@
       items:
         - name: Map control API classes
```
Comment (JuanPapas) [inline BingMaps/sdk-native/TOC.yml:75]: Maybe I'm not familiar enough with what the "TOC.yml" does or what role it plays in order to provide any relevant feedback.  Can you elaborate?

But that seems to be different from the actual API reference index page that corresponds to the documentation page below:
https://docs.microsoft.com/en-us/bingmaps/sdk-native/map-control-api/

**Review by JuanPapas — COMMENTED** [review]
Overall typo fixes for some of the existing pages look good.

Cc: @khass891891 (for awareness, since we need to port these changes back into the controls repo where these docs come from)

File: BingMaps/sdk-native/TOC.yml (hunk 1 above)
Comment (stevemunk) [reply BingMaps/sdk-native/TOC.yml:75]: image

Comment (JuanPapas) [reply BingMaps/sdk-native/TOC.yml]: Discussed offline and updated the Table of Contents as a result.

**Review by JuanPapas — APPROVED** [review]
Latest updates to avoid duplicated titles look good to me.

My recommendation would be to do the same for all main pages:

Can you also update the titles for two other main pages to keep them consistent?
"Map Control Concepts Reference - Native | Microsoft Docs"
"Map Services API Reference - Native | Microsoft Docs"

**Pull request merged**